webapp-django/crashstats/manage/jinja2/manage/status_message.html

```diff
@@ -25,7 +25,13 @@
                 <tr>
                     <td {% if not status.enabled %}class="strike" title="Disabled"{% endif %}>{{ status.message }}</td>
                     <td>{{ status.severity }}</td>
-                    <td><a href="{{ url('manage:status_message_disable') }}?id={{ status.id }}">Disable</a></td>
+                    <td>
+                        {% if status.enabled %}
+                        <a href="{{ url('manage:status_message_disable') }}?id={{ status.id }}">Disable</a>
+                        {% else %}
+                        disabled
```

Comment:
Not really, no. If there's something that needs to be said again, I guess you can just create a new message with the same content. But I don't expect problems to happen repeatedly. Or we're not doing a very good job... :) 

Also, that gives us a nice history of what happened on the website over time. You know, for when we're depressed.